src/tsconfig/index.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { EmptyTree, SchematicsException } from '@angular-devkit/schematics';
+import { Tree } from '@angular-devkit/schematics/src/tree/interface';
+import {
+  SchematicTestRunner,
+  UnitTestTree
+} from '@angular-devkit/schematics/testing';
+import { join } from 'path';
+import * as tsconfigSchema from './test/schema.json';
+import { TsConfigSchema } from './tsconfig.schema.js';
+import * as parameterSchema from './tsconfig.schema.json';
+
+import ajv = require('ajv');
+
+const collectionPath = join(__dirname, '../collection.json');
+
+describe('@co-it/schematics:tsconfig', () => {
+  const schemaValidator = new ajv();
+
+  const parameterDefaults: TsConfigSchema = {
```

Comment:
Read default values automatically from `.json` instead of assigning them manually.